modules/@angular/compiler/src/template_parser/template_parser.ts

```
@@ -675,7 +721,7 @@ class TemplateParseVisitor implements html.Visitor {
     const eventName = parts[1];
     const ast = this._parseAction(expression, sourceSpan);
     targetMatchableAttrs.push([name, ast.source]);
-    targetEvents.push(new BoundEventAst(eventName, target, ast, sourceSpan));
+    targetEvents.push(new BoundEventAst(eventName, target, null, ast, sourceSpan));
```

That would mean that everything else after it would also have a default value which would be misleading.